regolith/helpers/l_abstracthelper.py

```diff
@@ -0,0 +1,104 @@
+
+from regolith.helpers.basehelper import SoutHelperBase
+from regolith.fsclient import _id_key
+from regolith.tools import (
+    all_docs_from_collection,
+    get_pi_id,
+    get_person_contact
+)
+
+TARGET_COLL = "presentations"
+HELPER_TARGET = "l_abstract"
+
+def subparser(subpi):
+    subpi.add_argument(
+        "run",
+        help='run the lister. To see allowed optional arguments, type "regolith helper l_abstracts".')
+    subpi.add_argument(
+        "-v",
+        "--verbose",
+        action="store_true",
+        help="Increases the verbosity of the output.")
+    subpi.add_argument(
+        "-a",
+        "--author",
+        help='authors group ID(single argument only) to use to find presentation abstract.')
+    subpi.add_argument(
+        "-y",
+        "--year",
+        help='Start or end year of the presentation (single argument only) to use to find presentation.')
+    subpi.add_argument(
```

Comment:
I think I might search based on institution if it is a seminar or location if it is a meeting.  The best would be if location could take either.  if pres['type'] is seminar or colloquium it should look in pres['institution'] and if it is anything else, look in location.

Reply:
To double check, would it be preferential to have two command argument choice, one for '--location' and one for '--institution' or would it be easier to just use '--location' and from it, u can choose to enter an institution, city, state, etc. And either way, It will go into searching based on the type if its a seminar or colloquium or if not, look in location. I wasn't sure if it would confusing for the user to use '--location' for all types of locations such as institution since it's technically a location. In addition, I was thinking it would easier to have one that encompasses both types of locations so you don't have to remember to many arguments. I have figured out how to do it both ways, but let me know which may be more user-friendly and I will adjust it and push a test case for this condition. :)

Reply:
yes, just one.  Maybe we should call it `--loc_inst` and the help message should be very clear about what is wanted.